src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -93,13 +212,32 @@ export default function NurseryWithdrawals(props: NurseryWithdrawalsProps): JSX.
                 type='text'
                 className={classes.searchField}
                 iconRight='cancel'
+                value={searchValue}
+                onChange={(_id, value) => setSearchValue(value as string)}
               />
+              <NurseryWithdrawalsFiltersPopover filters={filters} setFilters={setFilters} organization={organization} />
+              <Grid xs={12} display='flex'>
+                {(Object.keys(filters) as (keyof typeof filters)[]).map(
+                  (filter: keyof NurseryWithdrawalsFiltersType) => {
+                    return filters[filter]?.map((value) => {
+                      return (
+                        <Pill
+                          key={value}
+                          filter={filter}
+                          value={filter === 'fromNurseryIds' ? getNurseryName(value) : value}
+                          onRemoveFilter={() => removeFilter(filter, value)}
+                        />
+                      );
+                    });
+                  }
+                )}
+              </Grid>
             </Grid>
             <Grid item xs={12}>
               <Table
                 id='withdrawal-log'
                 columns={columns}
-                rows={nurseryWithdrawals || []}
+                rows={searchResults || nurseryWithdrawals || []}
```

Comment:
we have a mix of latest search results vs cached memory results which can be inconsistent, ok to always use latest search results? which means if the filters are cleared - search for all nursery withdrawals again. Have one state to hold results.